### PR TITLE
fix: Rename Alert to Message Strip | Mark Alert as deprecated

### PR DESCRIPTION
**Pull request by salarenko**
## Related Issue
Closes SAP/fundamental-styles#579

## Description
This PR:
-  introduces new component `Message Strip` which codebase has been copied from `Alert` component

-  `Alert` component has been marked as Deprecated, it is still supported with fundamental-styles css classes

- Example demo with `Alert` has been removed in favor of `Message Strip` component since its the default component used to displaying alert messages

(reopened #611)

### CODE REVIEW

**Comment by salarenko** [comment]
@droshev Hey Deno, I've applied the changes we've been talking about.

File: src/overlay.scss [hunk 1]
```diff
@@ -11,6 +11,7 @@ $block: #{$fd-namespace}-overlay;
   $fd-overlay-background-color: rgba(#4a505c, 0.3) !default;
 
   --fd-overlay-background-color: #{$fd-overlay-background-color};
+  $fd-overlay-message-strip-top-margin: fd-space(7);
```
Comment (droshev) [inline src/overlay.scss]: Can you revert any changes in this file?

File: docs/_sass/_docs-display-component.scss [hunk 2]
```diff
@@ -112,6 +112,12 @@
                 margin-top: -410px;
             }
         }
+
```
Comment (droshev) [inline docs/_sass/_docs-display-component.scss]: can we remove this

File: docs/_sass/_docs-display-component.scss [hunk 3]
```diff
@@ -112,6 +112,12 @@
                 margin-top: -410px;
```
Comment (droshev) [inline docs/_sass/_docs-display-component.scss]: can we remove this as we agreed

File: src/overlay.scss (hunk 1 above)
Comment (salarenko) [reply src/overlay.scss]: These are the changes needed for messages strip to work.

File: docs/_sass/_docs-display-component.scss (hunk 2 above)
Comment (salarenko) [reply docs/_sass/_docs-display-component.scss]: We can't do that. This class is used in documentation when message strip is positioned inside overlay example.

**Pull request merged**